bitnami/redis-cluster/README.md

```diff
@@ -41,6 +49,46 @@ The main features of each chart are the following:
 
 ## Installing the Chart
 
+### Verify Kubernetes StorageClass
+
+Before installing the chart the Kubernetes StorageClass being used must have set `volumeBindingMode: WaitForFirstConsumer`.
+
+Check the (default) StorageClass or any other you intend to use.  
+
+```bash
+$ kubectl get storageclass -o wide
+```
+
+The default output below for a Kubernetes environment with digitalocean.com does not have the required StorageClass set with `VOLUMEBINDINGMODE` equal to `WaitForFirstConsumer`. 
```

Comment:
Could we make the description a bit more cloud-agnostic? We don't normally include explicit references to a cloud provider.